src/main/java/httpserver/Main.java

```diff
@@ -18,10 +21,11 @@ public static void main(String[] args) {
             try {
                 SocketServer socketServer = new HTTPSocketServer(new ServerSocket(5000));
                 String path = "/Users/molliestephenson/Java/Server/cob_spec/public";
+                String resources = "/Users/molliestephenson/Java/Server/HTTPServer/src/main/resources/form";
```

Comment:
This should be generic rather than hardcoded to your path. As it stands it wont work if I check this out as it will look for something under moliestephenson.